SilasKenneth/fast-food-fast-ui
# [Finishes #160136297] Mark orders as rejected or accepted

#### What this pull request does
Creates a page where an admin can accept or reject orders
##### Description of the task to be completed
 Create a page where an admin user can accept or reject orders
    -  Add links for the logged in user
    - Add links for the user to choose other options
##### Relevant pivotal tracker story
[#160136297](https://www.pivotaltracker.com/n/projects/2194200/stories/160136297)
##### How to test
  - Clone the repo ``git clone https://github.com/SilasKenneth/fast-food-fast-ui.git``
  -  Go to the fast-food-fast-ui directory and head to UI folder
  - Open the admin_orders.html file in your browser and that should work

#### Screenshot
